packages/website/pages/blog/categories/case_studies/ingrain-complaints/solutionsText.yaml

```diff
@@ -0,0 +1,7 @@
+id: solution
```

Comment:
This block can just go to the cardBlock file, and also rename that file to something more descriptive.